server/src/main/java/org/apache/druid/server/coordination/SegmentLoadDropHandler.java

```diff
@@ -88,9 +93,17 @@ public SegmentLoadDropHandler(
         config,
         announcer,
         segmentManager,
-        Executors.newScheduledThreadPool(
+        new ScheduledThreadPoolExecutor(
```

Comment:
This can remain as before:
```suggestion
        Executors.newScheduledThreadPool(
```